hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/token/TokenCommands.java

```diff
@@ -61,4 +61,5 @@ public boolean isVerbose() {
   public OzoneConfiguration createOzoneConfiguration() {
     return shell.createOzoneConfiguration();
   }
+
```

Comment:
We probably do not need this newline here.

Reply:
Sure.  I'll remove it if I need to change the patch for some other reason.

Reply:
+1 thanks